packages/manager/modules/billing/src/payment/method/add/index.html

```diff
@@ -136,9 +136,7 @@
                     data-ng-if="$ctrl.isStepVisible('bankAccountOwner')"
                 >
                     <div class="row">
-                        <div class="col-md-6">
```

Comment:
As you're deleting a `col-md-*`, you leave the above `row` with no correct children. You should remove the `row`as well.